server/svix-server/src/lib.rs

```diff
@@ -35,12 +38,14 @@ pub mod redis;
 pub mod v1;
 pub mod worker;
 
+#[tracing::instrument(name = "app_start", level = "trace")]
```

Comment:
```suggestion
#[tracing::instrument(name = "app_start", level = "trace", skipall)]
```
Same for the rest of the functions.

Also, I'm not quite sure why we would want to instrument these functions in the first place. I get instrumenting the migration one, but why the rest of these?